### ISSUE TITLE
SCRDS Server, Resource monitoring dies after ~1 hour

### ISSUE THREAD

**Issue by stigin** [pterodactyl/panel#343]
The resource monitoring for a SCRDS (GMOD) server lasts around 1 hour prior to the graph and status of the server becoming blank / red despite the server still being fully operational

![](http://i.imgur.com/R5qTDkO.png)

**Comment by DaneEveritt** [comment]
Does refreshing the page fix anything? I'm assuming by status becoming red you mean the little icon in the top left next to server name?

Did you make any edits to this server before this happened? I think this is caused by the same bug that results in duplicate socket.io connections which just breaks the panel a bit. Rebooting the daemon should fix everything.

**Comment by stigin** [comment]
Refreshing the page doesn't do anything.

Yup, that's correct i'm referring to the top left status icon.  

No edits have been made to the server prior to this happening. Rebooting the Daemon does fix the result for roughly 30 mins to 1 hour.

Below is the startup command we are using for GMOD.
-game garrysmod -console -port 27015 -strictportbind -norestart  +maxplayers 32 +map ttt_minecraft_b5 -console +gamemode terrortown -autoupdate

**Comment by DaneEveritt** [comment]
Thanks for the info, it might be a few days before I can get around to testing this more, but I will take a look. :)

**Comment by Celestialdeath99** [comment]
I have the issue with the Minecraft servers as well. After 1-2 hours the feed cuts off. I can still use the file manager, console and everything else but the stats cut out. I am able to fix it by putting the panel into maintenance mode then switching it back to productions mode.

**Comment by DaneEveritt** [comment]
Is this while the page is active, or just if the server runs for a few hours and you pull it up again in the panel it does this?

Also what browser, and OS for your computer. What OS for the daemon.

**Comment by Celestialdeath99** [comment]
This issue is both while the page is active, or if I just bring it back up later.

I have tried it with both Chrome and Microsoft Edge and Chrome on my Android phone.

I have used Ubuntu 16.10 and Windows 10 for my computer (Dual Boot). 

Both the panel and daemon are on Ubuntu 16.0.4.2 x64 servers. They are VM's and are separate machines. Both are on the same network and have been working. This issue was not present in the old 0.5.7 version. It is not a large deal, it is just annoying...
